Gopkg.toml

```diff
@@ -44,7 +44,7 @@
 
 [[constraint]]
   name = "github.com/zondax/ledger-cosmos-go"
-  version = "=v0.9.7"
+  version = "=v0.9.8"
```

Comment:
Diff between versions: https://github.com/ZondaX/ledger-cosmos-go/compare/v0.9.7...master

Reply:
As this library is getting more stable. Maybe it is time to transfer it to https://github.com/cosmos?

Reply:
Yes please

Reply:
What do we need for this? Isn't the https://github.com/cosmos/ledger-cosmos for this code?

Reply:
no. it is a bit more complicated:

- https://github.com/cosmos/ledger-cosmos      are two ledger apps (cosmos user, tendermint validator). Includes unit tests, build tools, etc. There two submodules because Ledger requires an specific repo format that does now allow for proper unit tests, etc.
    - https://github.com/tendermint/ledger-validator-app is the validator app prepared from Ledger
    - https://github.com/cosmos/ledger-cosmos-app is the cosmos app prepared from Ledger

- https://github.com/ZondaX/ledger-cosmos-go is the library to connect to the apps from go
- https://github.com/ZondaX/ledger-tendermint-rs is the library to connect to the apps from rust
- https://github.com/ZondaX/ledger-cosmos-js is the library to connect to the apps from js

:)

Reply:
I would prefer to transfer so repo urls are mapped by GitHub

Reply:
I have created an issue for the migration
https://github.com/cosmos/cosmos-sdk/issues/3681